cmd/commands/compose.go

```diff
@@ -59,11 +61,11 @@ func ConvertCommand(dockerCli command.Cli, options *composeOptions) *cobra.Comma
 				return err
 			}
 
-			j, err := template.JSON()
+			json, err := cloudformation.Marshall(template)
 			if err != nil {
```

Comment:
Slightly more idiomatic (to keep the `json` and `err` local;

```suggestion
			if json, err := cloudformation.Marshall(template); err != nil {
```